packages/api-derive/src/index.ts

```diff
@@ -15,6 +15,7 @@ import * as council from './council';
 import * as democracy from './democracy';
 import * as elections from './elections';
 import * as imOnline from './imOnline';
+import * as registrar from './registrar';
```

Comment:
I would probably put this under “parachains” rather - so functionality instead of the Substrate module